manifests/lvs/virtual_server.pp

```diff
@@ -116,6 +124,8 @@
   Optional[Integer[1]] $persistence_timeout = undef,
   Enum['TCP','UDP'] $protocol = 'TCP',
   Optional[Integer[1]] $quorum = undef,
+  Stdlib::Absolutepath $quorum_up = undef,
```

Comment:
I think `String[1]` was correct here.  Otherwise you can't follow the path to the script with any arguments.

Reply:
Do we are agreed to use Optional[String[1]] as datatype for quorum_up/quorum_down? (I also made a mistake in the test corrections)
Should I just revert my last commit?

Reply:
Or rebase to remove it.

Reply:
done